redhat-appstudio/service-provider-integration-operator
# remove port setting for webhooks

### What does this PR do?
Removes the setting of the webhook port from the controller manager options. Since we are not using webhooks, it is unnecessary and can be misleading.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-151

### How to test this PR?

Everything should work without change.

```
quay.io/redhat-appstudio/pull-request-builds:spi-controller-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
